2020-02-Spring/2_linked_lists/README.md

```diff
@@ -126,31 +147,117 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
-
-Source: TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
 #### Naive/Simple Solution
+The idea is to locate the sublist of nodes that sum to 0 and de-list them.<br>
+- Starting at the edging node, find lower-bound and upper-bound node of the sublist that sum to 0.
+- if found, de-list nodes from [LBound, UBound], including by re-link PreLBound->next = UBound->next;
+- If not found, move forward and check the next node.
```

Comment:
Are we checking the next node to see if it contributes to our zero sum, or are we starting a new outer loop iteration? Revise the phrasing here.
```suggestion
- Upon reaching the end of the list in this iteration of zero-sum searching, repeat the process beginning at the next node in the list.
```